tests/unit_test/test_curve.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+import pytest
+import unittest
+from beprof.curve import Curve
+
+
+class TestCurveInit(unittest.TestCase):
+    """
+    Testing Curve initialization and .x .y
+    """
+    def test_list_init(self):
+        simple_lists = [[-12, 1], [-1, 7], [0, 3], [3, 17]]
+        c = Curve(simple_lists)
+        assert list(c.x) == [e[0] for e in simple_lists]
```

Comment:
Use dedicated methods to compare lists or iterables, see:
https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertSequenceEqual
`assertListEqual`, `assertSequenceEqual `